### PR TITLE
Created nx_tutorial_script.py, nx_notebook.ipynb

### PR DESCRIPTION
**Pull request by sirichandana-v**
@MridulS Could you please check this PR and let me know if I should make any changes. In reference to #5

### CODE REVIEW

File: 2023-round-1/sirichandana-v/nx_tutorial_script.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+sp = dict(nx.all_pairs_shortest_path(DG))
+
+for key in sp:
+    for val in sp[key]:
+        l = len(sp[key][val])-1
+        if key != val:
+            print("Shortest Distance between {} and {} is {}".format(key, val, l))
+
```
Comment (MridulS) [inline 2023-round-1/sirichandana-v/nx_tutorial_script.py:21]: There is no need to find the path length, you just need to print out the path from one node to another.

Comment (sirichandana-v) [reply 2023-round-1/sirichandana-v/nx_tutorial_script.py:21]: oh..I will change that and print the path. Thank you.

Comment (sirichandana-v) [reply 2023-round-1/sirichandana-v/nx_tutorial_script.py]: @MridulS  I have updated the code. Could you pls check this and suggest me any other changes needed. I have also included #8 in this PR.

**Comment by sirichandana-v** [comment]
oh...noted...Didn't know that I need to create a different branch for having a different PR. Thank you.

**Comment by MridulS** [comment]
Looks good @sirichandana-v. Thanks!

**Pull request merged**